packages/twenty-front/src/modules/object-metadata/components/ObjectMetadataItemsLoadEffect.tsx

```diff
@@ -21,31 +21,28 @@ export const ObjectMetadataItemsLoadEffect = () => {
       skip: !isLoggedIn,
     });
 
-  const updateObjectMetadataItems = useRecoilCallback(
```

Comment:
refactored this hacky logic in something simpler

Reply:
Got it, I'll be on it